projects/packages/search/src/dashboard/components/record-meter/style.scss

```diff
@@ -39,17 +42,12 @@ ul.jp-search-chart-legend {
 .jp-search-notice-box {
 	background-color: white;
 	color: black;
-	border: lightgray 0.5px solid;
+	border: #dcdcde 0.5px solid; // grey #5
```

Comment:
You should be able to import these colours as Sass variables - like this:

```
@import "~@automattic/color-studio/dist/color-variables";

button {
  background: $studio-pink-50;
  color: $studio-white;
}
```

(from https://github.com/Automattic/color-studio/blob/master/README.md)

Reply:
Ah, thank you so much! I was wondering about this, so asked over in the P2 post but didn't think to ask it in here :) That will help a lot.